how-to/software/automatic-updates.md

```diff
@@ -341,6 +343,20 @@ The `unattended-upgrades` feature is helpful, does its job, and even sends out r
 An example of such a Fleet Management software for Ubuntu systems is [Landscape](https://ubuntu.com/landscape).
 
 
+(unattended_upgrades_postpone)=
+## Postponable updates
+By default, system updates are applied automatically in the background without any user interaction.
+Starting with Ubuntu 25.04, this behaviour can be changed to prompt active users before starting the update, allowing them to postpone the updates to a later time, up to a limited number of days as configured by the system administrator.
+
+To enable the feature, set the `Unattended-Upgrade::Postpone-For-Days` configuration in `/etc/apt/apt.conf.d/50unattended-upgrades` to a positive number, signifying the number of days that a user of this system is allowed to postpone the automatic updates for. Every time `unattended-upgrades` is run, after determining whether some updates are available to be applied it will prompt the user and shortly wait for an answer. If the user requests to postpone an update, the date and time of this action is recorded so that on future runs `unattended-upgrades` will only prompt the user again if the number of days elapsed from the first postpone request is lower than what is configured in `Unattended-Upgrade::Postpone-For-Days`.
+The feature is disabled if the number of days is set to 0.
+
+The `Unattended-Upgrade::Postpone-Wait-Time` configuration allows controlling the amount of time (in seconds) that a user of this system has available to send a postpone request after being prompted. If no postpone request is received within the specified time, then the updates will start being applied as normal.
+
+The prompting functionality is currently implemented graphically on Ubuntu Desktop by the `update-notifier` program. While the updates are being applied, an icon will also be visible in the system tray area informing the user so that they may know when it is safe to resume critical activities that may be affected by the updates.
+On other environments, you may implement your own prompting client by listening for the `AboutToStart` signal on the system bus and send a call to the `Postpone()` method. Read the `/usr/share/dbus-1/interfaces/com.ubuntu.UnattendedUpgrade.Pending.xml` interface specification for more details.
```

Comment:
```suggestion
On other environments, such as Ubuntu Server, you can implement your own prompting client by listening for the `AboutToStart` signal on the system bus and send a call to the `Postpone()` method. Read the `/usr/share/dbus-1/interfaces/com.ubuntu.UnattendedUpgrade.Pending.xml` interface specification for more details.
```

Reply:
Is that all the documentation that exists about this?

Reply:
Yes it is.